### PR TITLE
Use cookie if localStorage is not available (for safari in incognito mode)

### PR DESCRIPTION
**Pull request by ElForastero**
Added cookie micro-framework by MDN. It's similar to localStorage object and has almost the same methods.

Removed rollup and gulp form dependencies.

Built with babel.

If localStorage is not available cookie will be used.

### CODE REVIEW

**Comment by pbelyaev** [comment]
@ElForastero Tests are broken 😢

**Comment by ElForastero** [comment]
@pbelyaev Fixed. 👍

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  function VueLocalStorage() {
+    _classCallCheck(this, VueLocalStorage);
+
+    this.storage = _storage2.default;
+    //
+    // Object.defineProperty(this, "length", {
+    //   get() {
+    //     return this.storage.length;
+    //   }
+    // });
```
Comment (pbelyaev) [inline dist/index.js]: Why is this commented out?

**Review by pbelyaev — CHANGES_REQUESTED** [review]
Rebuild the dist file and we're okay.

**Comment by ElForastero** [comment]
@pbelyaev Done!

**Review by pbelyaev — APPROVED** [review]

**Pull request merged**